msteams-platform/sbs-meeting-token-generator.yml

```diff
@@ -207,6 +249,9 @@ items:
 
          :::image type="content" source="~/assets/images/meeting-token-generator/meeting-token-generator-appisready.png" alt-text="Screenshot shows the webpage that displays Your bot is ready.":::
 
+    > [!div class="nextstepaction"]
+    > [I ran into an issue](https://github.com/MicrosoftDocs/msteams-docs/issues/new?template=Doc-Feedback.yaml&title=%5BI+ran+into+an+issue%5D+Build+and+run+the+service&pageUrl=https%3A%2F%2Flearn.microsoft.com%2Fen-us%2Fmicrosoftteams%2Fplatform%2Fsbs-meeting-token-generator%3Ftabs%3Ddev%252Cvs%26tutorial-step%3D8&contentSourceUrl=https%3A%2F%2Fgithub.com%2FMicrosoftDocs%2Fmsteams-docs%2Fblob%2Fmain%2Fmsteams-platform%2Fsbs-meeting-token-generator.yml&documentVersionIndependentId=285d700c-17c0-24fe-d74e-fa717a812ce3&author=%40surbhigupta&metadata=%2A%2BID%253A%2Be473e1f3-69f5-bcfa-bcab-54b098b59c80%2B%250A%2A%2BService%253A%2B%2A%2Amsteams%2A%2A)
```

Comment:
Update title as

**Build and run the service - Visual Studio**

Reply:
Updated as suggested